src/dotnet-new3/Program.cs

```diff
@@ -66,13 +73,19 @@ private static DefaultTemplateEngineHost CreateHost(bool emitTimings)
 
             var builtIns = new AssemblyComponentCatalog(new[]
             {
-                typeof(RunnableProjectGenerator).GetTypeInfo().Assembly,            // for assembly: Microsoft.TemplateEngine.Orchestrator.RunnableProjects
-                typeof(NupkgInstallUnitDescriptorFactory).GetTypeInfo().Assembly,   // for assembly: Microsoft.TemplateEngine.Edge
-                typeof(DotnetRestorePostActionProcessor).GetTypeInfo().Assembly,    // for assembly: Microsoft.TemplateEngine.Cli
-                typeof(NupkgUpdater).GetTypeInfo().Assembly                         // for assembly: Microsoft.TemplateSearch.Common
+                // for assembly: Microsoft.TemplateEngine.Orchestrator.RunnableProjects
+                typeof(RunnableProjectGenerator).GetTypeInfo().Assembly,
+                // for assembly: Microsoft.TemplateEngine.Edge
+                typeof(NupkgInstallUnitDescriptorFactory).GetTypeInfo().Assembly,
+                // for assembly: Microsoft.TemplateEngine.Cli
+                typeof(DotnetRestorePostActionProcessor).GetTypeInfo().Assembly,
+                // for assembly: Microsoft.TemplateSearch.Common
+                typeof(NupkgUpdater).GetTypeInfo().Assembly
             });
 
-            DefaultTemplateEngineHost host = new DefaultTemplateEngineHost(HostIdentifier, HostVersion, CultureInfo.CurrentCulture.Name, preferences, builtIns, new[] { "dotnetcli" });
+            ConfigureLocale();
+
+            DefaultTemplateEngineHost host = new DefaultTemplateEngineHost(HostIdentifier, HostVersion, CultureInfo.CurrentUICulture.Name, preferences, builtIns, new[] { "dotnetcli" });
```

Comment:
`CultureInfo.CurrentUICulture.Name` is set to `Locale` property of the cost (read-write property). Should we change this property to be read only and return `CultureInfo.CurrentUICulture.Name`. I'm a bit concerned that we have multiple sources for locale: `CultureInfo.CurrentUICulture.Name`, `CultureInfo.CurrentCulture.Name`  and `ITemplateEngineHost.Locale` and they might get out-of sync.

Reply:
Or we can simply remove Locale property, not sure what's the benefit of it.

Reply:
Locale was always set to either `CultureInfo.CurrentCulture` or to "en-US" (mostly in tests). I don't see much point either. So I removed the locale from the host in the last commit. Please review with care as this maybe an impactful change.